ahfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>(I know it's in the inline comment, but maybe it would be worth moving it here under a sub-heading?): we should mention what the endstates are in terms of chemicalsystems, i.e. hydration free energy of ligand is ligand+water to water.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>thanks!</p>